client/client.go

```diff
@@ -364,8 +389,12 @@ func NewClientWithKeyspace(ctx context.Context, keyspaceID uint32, svrAddrs []st
 }
 
 // NewClientWithKeyspaceName creates a client with context and the specified keyspace name.
-func NewClientWithKeyspaceName(ctx context.Context, keyspace string, svrAddrs []string, security SecurityOption, opts ...ClientOption) (Client, error) {
-	log.Info("[pd] create pd client with endpoints and keyspace", zap.Strings("pd-address", svrAddrs), zap.String("keyspace", keyspace))
+func NewClientWithKeyspaceName(
+	ctx context.Context, keyspace string, svrAddrs []string,
```

Comment:
Is there possible that the keyspace name is an empty string?

Reply:
Yes, empty string, which is equal to "DEFAULT" default keyspace name internally, is a valid name.

Reply:
Committed the change "if keyspace is empty, fall back to the legacy API."